### DISCUSSION TITLE
How to disable "Memory Saver" with launch settings

### DISCUSSION THREAD

**Discussion by LameHonest** [hardkoded/puppeteer-sharp#2099] [Q&A] [answered]
When I restart the application the Memory Saver mode is always enabled by default. Can I disable it somehow using LaunchOptions?
![image](https://user-images.githubusercontent.com/81246949/222977708-682563a5-4bde-47df-b44a-7c8fd3fcc3d7.png)

**Comment by kblok** [comment]
I didn't find any command line argument to disable that. But we are sending several feature flags to disable background pages. Did you get any pages offline?

**Reply by LameHonest** [reply]
Yes, they did. The problem is that my application opens several tabs, but I don't go into them. They run in the background all the time and after a while the browser makes them offline.

**Comment by LameHonest** [comment, marked as answer]
Unfortunately, I did not find how this can be done through the command, but there is another way.
If you also need to create a new browser every time and do not manually turn off Memory Saver mode, then you need to change the registry.
Navigate to 'HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Chromium', if there is no such key, create one. Look for the key named HighEfficiencyModeEnabled or create a new one (DWORD 32-bit) and set the value to 0.
https://chromeenterprise.google/policies/#HighEfficiencyModeEnabled
